### ISSUE TITLE
Unable to share location using keyboard

### ISSUE THREAD

**Issue by andybalaam** [vector-im/element-web#20837]
### Steps to reproduce

1. In a room, with keyboard focus in the composer
2. Use the tab key to move the focus to the Location button and press Enter

### Outcome

The Share Location dialog opens, but I am unable to use the keyboard to navigate it or remove it.

I expected to be able to share my location using keyboard navigation.

If I click in the Share Location dialog with the mouse, I am able to use the keyboard to control it, but there is little to no visual feedback about where my keyboard focus is.

### Browser information

Firefox 96.0 (64-bit)

### URL for webapp

develop.element.io

### Application version

Element version: d731a2561933-react-afe6021c9a06-js-51f3fac87be4 Olm version: 3.2.8

### Homeserver

matrix.org

### Will you send logs?

No

**Comment by andybalaam** [comment]
This is fixed in the latest code. It is still impossible to drop a pin using the keyboard, but that is a different (and much harder) problem, so does not belong here.
